src/doc.rs

```diff
@@ -524,7 +524,17 @@ pub struct Lang([u8; 3], u8);
 impl Lang {
     pub const ARABIC: Self = Self(*b"ar ", 2);
     pub const BOKMÅL: Self = Self(*b"nb ", 2);
-    pub const CHINESE: Self = Self(*b"zh ", 2);
+
+    #[deprecated = "use Lang::SIMP_CHINESE instead"]
+    pub const CHINESE: Self = Self::SIMP_CHINESE;
+    /// Simplified Chinese.
+    // TODO: change to `zhs` for next breaking version
+    //       to distinguish from another Chinese:
+    //       `zht` (traditional Chinese)
```

Comment:
Same here, feel free to apply the change

Reply:
I'm not sure if it should be applied boldly – it may break documents using `#text(lang: 'zh')` :(